l10n-pocket/en/pocket/platforms.ftl

```diff
@@ -0,0 +1,36 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+### iOS URL: https://www-dev-pocket.allizom.org/ios/
+### iOS URL: https://www-dev-pocket.allizom.org/android/
+### iOS URL: https://www-dev-pocket.allizom.org/safari/
+### iOS URL: https://www-dev-pocket.allizom.org/chrome/
+### iOS URL: https://www-dev-pocket.allizom.org/opera/
+### iOS URL: https://www-dev-pocket.allizom.org/edge/
+### iOS URL: https://www-dev-pocket.allizom.org/welcome/
```

Comment:
@reemhamz Since the original about page was ported, the domain name has changed

Could you please update all `allizom.org` URLs to one of the following, please?

* www-dev-pocket.allizom.org (Dev) -> dev.tekcopteg.com
* www-pocket.allizom.org (Stage) -> tekcopteg.com

Reply:
@stevejalim since I just have the Dev URLs, i'll just change them to `dev.tekcopteg.com`. However I don't see any of the Pocket staging URLs anywhere in the code, is there somewhere you've seen it in particular? Just want to double check with you in case.

Reply:
@reemhamz Yeah I think there are no stage URLs in there after all. Thanks for the diligence!